Taskfile.yml

```diff
@@ -24,6 +24,8 @@ tasks:
       - go install github.com/boumenot/gocover-cobertura@latest
       - go install mvdan.cc/gofumpt@latest
       - go install github.com/goreleaser/goreleaser@latest
+      - go get github.com/bats-core/bats-core@latest
+      - bash $(go env GOMODCACHE)/$(go list -m github.com/bats-core/bats-core | tr ' ' @)/install.sh ./test/integration/vendor/bats
```

Comment:
The only thing I don't think I like about this approach is the changing of the vendor'ed files when running `task unit` or something.  It's not really idempotent and could be confusing, but on the flip side, it would hopefully ensure the developer make sure to vendor in the changes if/when bats chagnes.